### ISSUE TITLE
[Android] Yuzu shows up in google play update and cannot be removed

### ISSUE THREAD

**Issue by InnocentAngel** [yuzu-emu/yuzu#12942]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

All Builds

### Description of Issue

I have Yuzu Android Github downloaded. Google play store detects yuzu downloaded and says there is an update. Its annoying to always show that theres an update available even though I have the github version downloaded and up to date. Another annoying issue is i cant refresh the updates in google play when yuzu is always there. I have to close and relaunch app to "refresh" the app updates. 

### Expected Behavior

I would expect yuzu github android to have some option to remove it from play store updates or not be detected by play store.

### Reproduction Steps

Download yuzu github android and itll show there is an update in google play even though its github.

### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/14188614/yuzu_log.txt.old.txt)

### System Configuration

CPU:SD 8 Gen 2
GPU/Driver:Adreno 740
RAM:8 Gb
OS: Android 14

**Comment by esQmo** [comment]
Google Play Store can update an app that was installed from a third-party source, as long as the app is also available on the Google Play Store and the installed app meets some criteria.

**Comment by Pergmen** [comment]
This is an incredibly annoying problem even for users who already have auto updates disabled. The play store and github builds can't even update off of one another, so they may as well have different package names. Otherwise, the only user-end solution to this would require root access

**Comment by iamnotkurtcobain** [comment]
Yeah this sucks. But it's Google's fault. They changed something in the Play Store.

**Comment by tommy-sorce** [comment]
Yep. That's really bothering
